DependencyInjection/Configuration.php

```diff
@@ -725,6 +729,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->ifTrue(function ($v) { return 'sentry' === $v['type'] && null !== $v['hub_id'] && null !== $v['client_id']; })
                 ->thenInvalid('You can not use both a hub_id and a client_id in a Sentry handler')
             ->end()
+            ->validate()
+                ->ifTrue(function ($v) { return 'sentry_breadcrumb' === $v['type'] && empty($v['sentry_handler']); })
```

Comment:
I think `$v['sentry_handler']` is always defined, and `null` by default.
```suggestion
                ->ifTrue(function ($v) { return 'sentry_breadcrumb' === $v['type'] && !$v['sentry_handler']; })
```

Reply:
Yes, you're right, fixed.